cli/src/main/resources/nls/Help_de.properties

```
@@ -95,6 +95,8 @@ cmd.shell.detail=Die integrierte Shell bietet Konsolenbenutzer*Innen eine neue B
 cmd.sonar=Werkzeug Kommando für SonarQube.
 cmd.sonar.detail=SonarQube ist eine Plattform für die kontinuierliche Code-Qualitätsprüfung. Detaillierte Dokumentation ist zu finden unter https://docs.sonarqube.org/
 cmd.sonar.val.command=Auszuführende Aktion (START|STOP|ANALYZE)
+cmd.status=Checkt deinen IDE-Status
+cmd.status.detail=Um deinen IDE-Status zu checken, geben sie einfach 'ide status' in die Konsole ein.
```

Würde das evtl. detaillierter beschreiben. Was genau gibt status aus?
```suggestion
cmd.status.detail=Um den Status von IDEasy zu überprüfen, geben Sie einfach 'ide status' in die Konsole ein.
```

---

done